Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,6 +489,35 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    updateAntialiasing(): void {
+      if (this._object.getAntialiasing() != 'None') {
```

Comment:
Always use strict inequality
```suggestion
      if (this._object.getAntialiasing() !== 'None') {
```